ara/views.py

```diff
@@ -74,17 +74,30 @@ def playbook(playbook):
 
 
 @app.route('/playbook/<playbook>/host/<host>')
-def playbook_host(playbook, host):
+@app.route('/playbook/<playbook>/host/<host>/status/<status>')
```

Comment:
I think that we should probably be using query parameters for filtering, as probably a more flexible and REST-ish type of solution.  E.g., `/playbook/playbookid?status=ok&host=localhost`.  This opens up the possibilities for more complex filters (`?status=ok&status=changed`).